custom/enikshay/private_sector_datamigration/factory.py

```diff
@@ -120,6 +126,12 @@ def get_person_case_structure(self):
 
         kwargs['attrs']['update']['facility_assigned_to'] = self._location_owner_id
 
+        if self.beneficiary.pincode:
+            kwargs['attrs']['update']['current_address_postal_code'] = self.beneficiary.pincode
+
+        if self.beneficiary.villageTownCity is not None:
+            kwargs['attrs']['update']['current_address_village_town_city'] = self.beneficiary.villageTownCity
```

Comment:
for the legacy_* properties, I'd like to explicitly mark them as null if that's the case, for my own record keeping.

For the current_address_* properties, I like to follow the convention that if they weren't set in UATBC, then the case should look like they were never entered in the app.  But it's a somewhat arbitrary convention.